src/components/sidebar/Ship.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ShipInterface } from '../../context/PlayerShipsContext';
+import Cell from '../Cell';
+
+const Ship: React.FC<ShipInterface> = function ({ name, shipSections }) {
+    const cellMap = shipSections.map((section, idx) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <Cell shipName={name} row={0} column={0} isHit={false} isShip key={name + idx} />
```

Comment:
Is passing all of the other props besides `isShip` necessary? Currently, the `Cell` component only takes in that one `isShip` prop.

Reply:
Not all of those are necessary right now, but they will be in the future when we add the hit-ships functionality. We have made the row and column fields optional though, since only the cells on the board need those.